types/twine-sugarcube/jquery.d.ts

```diff
@@ -0,0 +1,30 @@
+import { Passage } from "./passage";
+
+declare global {
+	namespace JQuery {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/jquery.d.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.